data/json/zones.json

```diff
@@ -15,8 +15,8 @@
   {
     "id": "zone_unload_all",
     "type": "LOOT_ZONE",
-    "name": "Loot: Unload Containers",
+    "name": "Loot: Unload Everything",
     "can_be_personal": true,
-    "description": "Any containers in this zone you don't otherwise sort will have their contents dumped out and placed on the same tile.  This includes corpses."
+    "description": "Any containers in this zone you don't otherwise sort will have their contents dumped out and placed on the same tile.  This includes corpses, magzines, and magazine wells."
```

Comment:
```suggestion
    "description": "Any containers in this zone you don't otherwise sort will have their contents dumped out and placed on the same tile.  This includes corpses, magazines, and magazine wells."
```